src/cassy.py

```diff
@@ -50,15 +54,18 @@ def get_env_data(node_id, env_variable):
         )
 
         if not rows:
-            raise Exception('Invalid Environmental Variable or Node ID')
+            raise Exception('No environmental data found for ' + node_id + ' and ' + env_variable + '.')
```

Comment:
Not a good idea to raise Exception (I can explain this if needed but it seems like you know and just missed a couple)